modelci/types/models/common.py

```diff
@@ -262,3 +262,31 @@ class Config:
             DataType: named_enum_json_encoder,
             ModelInputFormat: named_enum_json_encoder,
         }
+
+
+class ModelVersion(object):
```

Comment:
Is the `ModelVersion` class only needed to generate the model path, Is it necessary?

Reply:
This `ModelVersion` in `types/models/common.py` is a copy of the one in `types/bo/model_objects`. It is like Task, Frameworks, etc. I make this copy for the form consistency. For now types in` common.py` has not replaced old `bo` types in many modules, so this copy is not referred elsewhere. While any further development is recommended to  use this `ModelVersion` rather than the old one.

Reply:
Current ModelVersion is replaced by a simple int field. You may check https://github.com/cap-ntu/ML-Model-CI/blob/master/modelci/types/models/mlmodel.py#L72